### PR TITLE
Added Angelic Block height limiter

### PR DESCRIPTION
**Pull request by Sefiraat**
## Description
Added height limits to the Angelic Block's placement. Previously blocks were 'placable' outside of world bounds. While the block wouldn't actually appear, the placement would be remembered and particles would continue to spawn without a way for server owners to remove without directly editing config.

## Changes
Added a check for the world height before the block placement is attempted.

## Checklist
- [x] I have fully tested the proposed changes and promise that they will not break everything into chaos.
- [x] I followed the existing code standards and didn't mess up the formatting.
- [ ] I did my best to add documentation to any public classes or methods I added.
- [ ] I have added `Nonnull` and `Nullable` annotations to my methods to indicate their behaviour for null values

### CODE REVIEW

File: src/main/java/io/github/thebusybiscuit/sensibletoolbox/blocks/AngelicBlock.java [hunk 1]
```diff
@@ -81,6 +81,11 @@ public void onInteractItem(PlayerInteractEvent event) {
         event.setCancelled(true);
     }
 
+    private boolean isWithinWorldBounds(Block b) {
```
Comment (TheBusyBiscuit) [inline src/main/java/io/github/thebusybiscuit/sensibletoolbox/blocks/AngelicBlock.java:84]:
```suggestion
    private boolean isWithinWorldBounds(@Nonnull Block b) {
```

File: src/main/java/io/github/thebusybiscuit/sensibletoolbox/blocks/AngelicBlock.java [hunk 2]
```diff
@@ -81,6 +81,11 @@ public void onInteractItem(PlayerInteractEvent event) {
...
+        Location loc = b.getLocation();
+        return loc.getY() > loc.getWorld().getMinHeight() && loc.getY() < loc.getWorld().getMaxHeight();
```
Comment (TheBusyBiscuit) [inline src/main/java/io/github/thebusybiscuit/sensibletoolbox/blocks/AngelicBlock.java:86]: `getMinHeight()` does not exist on older versions.

**Review by TheBusyBiscuit — CHANGES_REQUESTED** [review]

File: src/main/java/io/github/thebusybiscuit/sensibletoolbox/blocks/AngelicBlock.java (hunk 2 above)
Comment (Sefiraat) [reply src/main/java/io/github/thebusybiscuit/sensibletoolbox/blocks/AngelicBlock.java]: Thanks, sorted and retested

**Review by TheBusyBiscuit — APPROVED** [review]
LGTM

**Pull request merged**